### DISCUSSION TITLE
Investigate moving from LightGallery to Spotlight

### DISCUSSION THREAD

**Discussion by rprieto** [thumbsup/thumbsup#277] [Q&A]
LightGallery works very well, but the license has changed from Apache to GPL in 2016.
This means we cannot update from v1.2 to newer versions.

Spotlight (https://github.com/nextapps-de/spotlight) seems to be mostly equivalent, doesn't require jQuery, and is licensed under Apache.

If staying on v1.2 of LightGallery starts causing issues, we could migrate to Spotlight in future versions of each theme.

**Comment by dravenst** [comment]
Agreed, the current version of lightGallery is pretty old now and I noticed has some issues with video autoplay.  Spotlight looks pretty good.  Thanks!
